src/interfaces/api-filing-interface.ts

```diff
@@ -87,12 +87,9 @@ export interface ApiFilingIF {
 
     // consent to continuation out filings only
     consentContinuationOut?: {
+      country: string
       expiry: IsoDatePacific
-      orderDetails: string
-      foreignJurisdiction: {
-        country: string
-        region?: string
-      }
```

Comment:
This interface is for filings coming back from the Legal API. (Unfortunately they do not match the schema for filings sent _to_ the Legal API.)

Example:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/deae5aab-2956-45fc-8176-783bc657831b)